poll/poll.py

```diff
@@ -287,8 +289,8 @@ class PollBlock(PollBase):
     )
```

Comment:
The default values above could be translated too to make this more understandable to new users. ('Poll', 'What is your favorite color?', 'Red', 'Blue', 'The answer options on this poll.', etc.)